metal/roles/pxe_server/defaults/main.yml

```diff
@@ -1,3 +1,4 @@
 iso_url: "https://download.rockylinux.org/pub/rocky/8.6/isos/x86_64/Rocky-8.6-x86_64-minimal.iso"
 iso_checksum: "sha256:a9ece0e810275e881abfd66bb0e59ac05d567a5ec0bc2f108b9a3e90bef5bf94"
 timezone: Asia/Ho_Chi_Minh
+dhcp_proxy: false
```

Comment:
I think we can enable DHCP proxy by default because most people have a standard home network with existing DHCP server (backward compatibility is not a problem at the moment).

Reply:
Sounds good, I'll switch some of the terms and configure scripts around for this.